outbound-dialer/src/main/java/org/motechproject/nms/outbounddialer/service/TargetFileNotification.java

```diff
@@ -0,0 +1,55 @@
+package org.motechproject.nms.outbounddialer.service;
+
+/**
+ * Request body
+ *
+ * 4.3.1 TargetFile Notification API
+ * NMS shall invoke this API to notify IVR platform when a target file is ready.
+ * http://<IVROBDAPI:port>/obdmanager/notifytargetfile
+ */
+public class TargetFileNotification {
```

Comment:
This is not s service class, just a DTO used to send in the POST body to the IVR system